package.json

```diff
@@ -34,15 +34,17 @@
   ],
   "contributors": [
     {
-      "name": "Tim Levett",
```

Comment:
`contributors` is a good call.  He doesn't at this point have publish privs to npm for the project.